server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadGet.feature

```diff
@@ -21,10 +21,9 @@ Feature: Download GET
     Given "username@domain.tld" mailbox "inbox" contains a message "1" with an attachment "2"
     When "username@domain.tld" downloads "2" with "myFileName.txt" name
     Then the user should receive that attachment
-    And the attachment is named "myFileName.txt"
```

Comment:
According to this discussion, we should add the charset in the content-disposition header:
http://stackoverflow.com/a/20933751